test/functional/feature_nulldummy.py

```diff
@@ -47,6 +47,7 @@ def set_test_params(self):
             f'-segwitheight={COINBASE_MATURITY + 5}',
             '-addresstype=legacy',
         ]] * 2
+        self.rpc_timeout = 480
```

Comment:
why is this needed? A common thing in our tests is to generate 100 blocks. If this times out, the default should be bumped instead of the overwrite for each test.